sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/operator/date/BeamSqlTimestampMinusIntervalExpression.java

```diff
@@ -52,8 +50,8 @@ public boolean accept() {
   static boolean accept(List<BeamSqlExpression> operands, SqlTypeName outputType) {
     return operands.size() == 2
         && SqlTypeName.TIMESTAMP.equals(outputType)
```

Comment:
I'm not sure this is actually correct. I think this should be something more like: `operands.get(0).getOutputType().equals(outputType)`. That is to say, the input DATETIME type is also the output type.